code/modules/cm_marines/smartgun_mount.dm

```diff
@@ -128,15 +132,12 @@
 
 /obj/machinery/m56d_post/examine(mob/user)
 	..()
-	if(!anchored)
-		to_chat(user, "It must be <B>screwed</b> to the floor.")
-	else if(!gun_mounted)
+	if(!gun_mounted)
 		to_chat(user, "The <b>M56D Mounted Smartgun</b> is not yet mounted.")
```

Comment:
`to_chat(user, "The <b>M56D Mounted Smartgun</b> is not yet mounted.")`

`to_chat(user, "The <b>[src]</b> is not yet mounted.")`